src/utils/layer-manager-utils.js

```diff
@@ -46,7 +46,7 @@ export const layerManagerOrder = (datasets, activeLayers, callback) => {
   callback({ activeLayers: updatedLayers });
 };
 
-export const createLayer = (layer, map) => {
+export const createLayer = (layer) => {
```

Comment:
I'd prefer the spinner not to show up when removing a layer. I liked Marta's explanation and it makes sense to me--> "...removing a layer we're not showing a spinner as it happens instantly)" otherwise it can be overwhelming.

Reply:
Yeah I agree, indeed my argument was that we might not need it at all when adding `Biodiversity` layers, because it is quite fast and the user already is seeing that something is happening because the layer is being painted (something that does not happen when adding the `Human pressures` rasters, as some computation needs to be done after `clicking` the button to add the layer).
Does this make sense?
My point is: I would remove the spinner on `Biodiversity` layers creation

Reply:
Agree with you @weberjavi

Reply:
That's true, biodiversity layers are rendering way faster than the other ones, we can remove the spinner there. It's a bit slower when you choose from the Fine Scale Data but still, there is something going on the screen - going to the location, rendering tiles.